v2/ruby/README.md

```diff
@@ -0,0 +1,34 @@
+# AdSense Management API Ruby Samples
+
+This is a set of simple samples written in Ruby, which provide a minimal
+example of Google AdSense integration within a command line application.
+
+This starter project provides a great place to start your experimentation into
+the AdSense Management API.
+
+## Prerequisites
+
+Please make sure that you're running Ruby 2.5+ and you've run
+`gem install` in this directory to install all prerequisites.
```

Comment:
```suggestion
`bundle install` in this directory to install all prerequisites.
```

Reply:
Or alternatively `gem install google-apis-adsense_v2`

Reply:
Going with the second option.